test/metabase/driver/druid_test.clj

```diff
@@ -59,3 +62,123 @@
 (expect-with-engine :druid
   :completed
   (:status (process-native-query native-query-2)))
+
+
+;;; +------------------------------------------------------------------------------------------------------------------------+
```

Comment:
no Druid tests are separate because we use a different test data, Druid is a timeseries (i.e., single-table) DB